astropy/coordinates/angles/core.py

```diff
@@ -11,11 +11,12 @@
 import numpy as np
 
 from astropy import units as u
+from astropy.units.quantity import SpecificTypeQuantity
```

Comment:
Just import from `astropy.units` itself (explicit import is nice).

Reply:
Would this particular case be solved if `astropy/units/__init__py` had an `__all__`? If not, that would seem a bug in VSCODE - it should obviously import from as high a level as possible (or maybe, equivalently, using minimum distance from where the import is done).

Reply:
I'm not sure exactly how it resolves imports. But I imagine PEP-8 strongly informs its strategy. Almost certainly `__all__` in `units/__init__.py` will help, or at the very least not hurt.